docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+hypthosis:
+  build: .
+  ports:
+   - "5000:5000"
+  environment:
+    DATABASE_URL: "postgresql://hypothesis:mysecretpassword@localhost:5432/hypothesis" #username:password@host:port/database
+    #APP_URL: http://127.0.0.1  #Replace with IP or DNS Name
+    SECRET_KEY: TCtkLQ3k5kNkezsFiPDvq04yG4olCwU6QF7XT6gktGDaEiBZAnxkDJXYCLMfF96W1j+A1PMO+vyfQUgMboEnoQ
+    #ALLOWED_ORIGINS: "*" #origins allowed to connect over the WebSocket protocol
+    #CLIENT_ID:        #a unique API key for authentication
+    #CLIENT_SECRET: #a unique API secret for signing authentication requests
+    #ELASTICSEARCH_INDEX: #the Elasticsearch index name for annotation storage
+    #MAIL_DEFAULT_SENDER: #a sender address for outbound mail
+    #WEBASSETS_BASE_DIR: #the base directory for static assets
+    #WEBASSETS_BASE_URL: #the base URL for static asset routes
+  links:
+   - redis
+   - postgres
+   - elasticsearch
+  command: python -m smtpd -n -c DebuggingServer localhost:25
```

Comment:
Perhaps I'm missing something, but this means that you're not running Hypothesis at all. I was suggesting earlier that you might want to run the debugging server in another container, but probably not in the hypothesis container.